node-webservice/react-client/src/Components/HomePage.jsx

```diff
@@ -7,7 +7,13 @@ const HomePage = props => (
         <h1>Join a game</h1>
         <JoinGameForm {...props}/>
         <br/>
-        <Link to="/host">Host a game</Link><br/>
+        <form>
+            <button type="submit" formaction="/host" 
+            style={{fontSize: 'calc(10px + 2vmin)', padding: '5px 10px'}}>
+                Host a Game
```

Comment:
<img width="317" alt="Screen Shot 2020-04-01 at 11 47 13 AM" src="https://user-images.githubusercontent.com/11395364/78174737-96683f80-740e-11ea-9bb2-46d3c9ea81c8.png">

Design wise, I'm a bit unsure about making the host button bigger. It has similar size on the page as the join form, but also no clear seperation from the join form, which could make it look like it's part of the join form. 

I'm not against making the 'host a game' section larger on the home page, but I think a more consistent design would be better, such as an initial page with two buttons 'join a game' and 'host a game', which lead to either the join form or Unity game embed, or two clearly seperate sections on the page for joining and hosting the game